Dockerfile

```diff
@@ -0,0 +1,14 @@
+### 1. Get python 3.6
+FROM python:3.6-alpine
```

Comment:
I'd prefer Python-3.7 as that's what we have easily available on our Macs for dev purposes.

I'd also prefer it we could move to using distroless images if possible but that might be for a different ticket as I don't know quite how hard it is. For this container it may be as easy as copying the pattern at https://github.com/GoogleContainerTools/distroless/issues/171#issuecomment-375297901

Reply:
Our emr  has python version 3.6 so thought have same version  for unit testing

Reply:
That makes far more sense for the Python version. And yes, distroless doesn't makes less sense for this as it needs both Python & JDK. We *may* be able to make it work but it's probably not as trivial as a python-only container.